ComponentKit/Core/Scope/CKComponentScopeHandle.mm

```diff
@@ -56,7 +58,7 @@ - (instancetype)initWithListener:(id<CKComponentStateListener>)listener
                  rootIdentifier:rootIdentifier
                  componentClass:componentClass
                           state:initialStateCreator ? initialStateCreator() : [componentClass initialState]
-                     controller:newController(componentClass)];
+                     controller:nil];
```

Comment:
Good call

On Thursday, August 18, 2016, Eric Czarny <notifications@github.com> wrote:

> In ComponentKit/Core/Scope/CKComponentScopeHandle.mm
> <https://github.com/facebook/componentkit/pull/605#discussion_r75315880>:
>
> > @@ -56,7 +58,7 @@ - (instancetype)initWithListener:(id<CKComponentStateListener>)listener
> >                   rootIdentifier:rootIdentifier
> >                   componentClass:componentClass
> >                            state:initialStateCreator ? initialStateCreator() : [componentClass initialState]
> > -                     controller:newController(componentClass)];
> > +                     controller:nil];
>
> I think it may be worth adding a quick comment here explaining that the
> controller will be created once the component acquires the scope handle.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/componentkit/pull/605/files/d52b88de7f03e65902d0d78d746b75756d1c9e44#r75315880>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA0WQYsbKlWl3BEGHZehP3Ff7BKdNEDSks5qhGmkgaJpZM4JmyKy>
> .
>